Document/0x06a-Platform-Overview.md

```diff
@@ -48,35 +48,33 @@ Signing application code in iOS is different than in Android. In the latter you
+Building encryption into the physical architecture makes it a default security control that is able to encrypt all data stored on an iOS device. As a result, data protection is implemented at software level and is enhanced by the hardware and firmware encryption to provide a greater degree of security.
 
-Building encryption into the physical architecture makes it a default security control that is able to encrypt all data stored on an iOS device. As a result data protection is implemented at the software level and works with the hardware and firmware encryption to provide a greater degree of security.
-
-When data protection is enabled, each data file is associated with a specific class that supports a different level of accessibility and protects data based on when it needs to be accessed. The encryption and decryption operations associated with each class are based on multiple key mechanisms that utilizes the device's UID and passcode, plus a class key, file system key and per-file key. The per-file key is used to encrypt the file content. The class key is wrapped around the per file key and stored in the file's metadata. The file system key is used to encrypt the metadata. The UID and passcode protect the class key. This operation is invisible to users. For a device to utilize data protection, a passcode must be used when accessing that device. The passcode does not only unlock the device, but also combined with the UID creates iOS encryption keys that are more resistant to hacking efforts and brute-force attacks. It is with this that users need to enable passcodes on their devices to enable data protection.
+When data protection is enabled, each data file is associated with a specific class that supports a different level of accessibility and protects data based on when it needs to be accessed. The encryption and decryption operations associated with each class are based on multiple key mechanisms that utilizes the device's UID and passcode, plus a class key, file system key and per-file key. The per-file key is used to encrypt the file content. The class key is wrapped around the per file key and stored in the file's metadata. The file system key is used to encrypt metadata. The UID and passcode protect the class key. This operation is invisible to users. Passcode must be used when accessing the device to enable data protection. The passcode does not only unlock the device, but also combined with the UID creates iOS encryption keys that are more resistant to hacking efforts and brute-force attacks. Enabling data protection is the main reason for the users to use passcodes on their devices.

```

Comment:
> The file system key is used to encrypt metadata.

Not any metadata, *the* metadata.

> Passcode must be used

I think still still needs an article, i.e. "a" or "the".

> Enabling data protection is the main reason for the users to use passcodes on their devices.

I think you could just say "for users" here instead of "for the users", as it talks about all  users in general.